main/src/resources/benchmark/BenchmarkSet.flix

```diff
@@ -0,0 +1,126 @@
+namespace BenchmarkSet {
+
+    ///
+    /// create a set of integers and compute the size of the result.
+    ///
+    @benchmark
+    pub def benchmark01(): Bool = Set.size(List.toSet(List.range(0, 75))) `assertEq!` 75
```

Comment:
We do not use assertEq any more. Its deprecated. Simply return the boolean.

Reply:
> As expected, correct? :)
Yes, there's a lot of list operations under the hood, so I guess it was to be expected. My only concern is whether the small number of elements will make it hard to compare it against future, faster implementations.But I guess we'll cross that bridge when we come to it

> We do not use assertEq any more. Its deprecated. Simply return the boolean.
Roger that, will refactor.